### PR TITLE
REVIEW: XMPP Newsletter April 2021

### PR DESCRIPTION
**Pull request by Echolon**
### Final release is planned for **5th of May 2021**.
---
You can make a simple comment or create a merge request to make things easy going for review. You don't need any special technical knowledge to contribute. If you need help, please ask for help in the Communication-Team MUC: commteam@muc.xmpp.org

- [You can use our online Pad](https://yopad.eu/p/xmpp-newsletter-365days)! Please be tidy and write something instead of dropping bare HTTP links :pray: 
- Just paste your prepared text in this thread below or in our XMPP channel (commteam@muc.xmpp.org)
- [Make a pull request by editing the file in this repository](https://github.com/xsf/xmpp.org/blob/newsletter-april2021/content/posts/newsletter/2021-05-05.md)

:arrow_right: Everyone participating, please do not just drop bare links - rather try to make a short and decent formulated description (~5 lines) what is the main outcome of your source. You do not need to write formal - be passionate why this motivates you or what you work on! 

**Note: If you create a pull request, which is highly appreciated, please @mention me once your PR is created!** :pray: 

---

This is our planed timeline for this draft:
XMPP Newsletter Timeline

Status:
- [ ] Collection of news
- [ ] Review
- [ ] Blog post + Tiny Newsletter
- [ ] Social media
- [ ] Translations
- [ ] Archived ([Use pandoc!](https://pandoc.org))

### CODE REVIEW

File: content/posts/newsletter/2021-05-05.md [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+
+![Gajim chat view](/images/newsletter/2021-04/gajim_chat-view-april.png "Gajim chat view")
+
+Libervia (formerly Salut à Toi) has been selected for a grant by NLNet/NGI0 Discovery Fund (with financial support from European Commission's Next Generation Internet programme) for an [ActivityPub <=> XMPP 
+gateway doubled with Pubsub end-to-end encryption project](https://www.goffi.org/b/activitypub-gateway-and-pubsub-e2ee-QGqK). The XMPP <=> ActivityPub gateway will join two major open and decentralised protocols. In practice it will be a XMPP server component (usable with any server), and implement the ActivityPub server to server protocol (or "Federation Protocol"). On XMPP side, it will be mostly a Pubsub service (with some extra, like private messages converted to XMPP message stanza).
+
+[Vaxbot continues to grow](https://monal.im/blog/vaxbot-continues-to-grow/): VaxBot is a free service that helps you find vaccine appointments in your state or country if available. During the last weeks, [Vaxbot](vaccine.monal.im) continued to grow. The service just hit 8 million messages sent and it sends around 800,000 messages a day. XMPP and
```
Comment (therealjeybe) [inline content/posts/newsletter/2021-05-05.md:52]: maybe add an "https://" here - is a little more beautiful
shouldn't make a difference in the generated html, but in github preview it messed the link up due to the missing protocol

File: content/posts/newsletter/2021-05-05.md [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+
+Snikket announces that [XMPP Account Portability](https://snikket.org/blog/dapsi-fund-account-portability/) was funded under the [NGI DAPSI](https://dapsi.ngi.eu/) initiative. It will cover the needed standards (beyond older [XEP-0227](https://xmpp.org/extensions/xep-0227.html) and [XEP-0238](https://xmpp.org/extensions/xep-0283.html)), open-source tools and integrating said functionality into Snikket.
+
+Nicola Fabiano published an article titled: [Whatsapp? No thanks, I prefer to have control over my data](https://www.nicfab.it/whatsapp-no-thanks-i-prefer-to-have-control-over-my-data/)
+
+## Software news
+
+### Clients and applications
+
+[Conversation 2.9.11 and 2.9.12 have been released](https://github.com/iNPUTmice/Conversations/blob/master/CHANGELOG.md), fixing 'No Connectivity' issues on Android 7.1 and adding support roster pre-authentication.
```
Comment (therealjeybe) [inline content/posts/newsletter/2021-05-05.md:67]: adding support **for** roster pre-authentication?

File: content/posts/newsletter/2021-05-05.md [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+
+[Gajim 1.3.2 has been released](https://gajim.org/post/2021-04-24-gajim-1.3.2-released/): This release brings back translations for Windows users. It also adds some small fixes and improvements.
+
+[Monal 5 Beta 4 has been released](https://monal.im/blog/monal-5-beta-4/) with lots of updates.
+
+Since there appeared a lot of [sendxmpp](https://sendxmpp.hostname.sk/) alternatives over the last years there is now an work-inprogress overview in the [XSF wiki](https://wiki.xmpp.org/web/User:MDosch/Sendxmpp_incarnations).
+
+[UWPX is replacing the SQLite-net DB backend with a new Entity Framework Core DB](https://github.com/UWPX/UWPX-Client/releases/tag/v.0.31.0.0). This comes with a fix and update of the OMEMO implementation to the latest version of the draft (OMEMO 0.7.0 (2020-09-05)). This version is incompatible with all other previous versions of OMEMO and UWPX allows you now to only exchange OMEMO encrypted messages with contacts that support at least OMEMO 0.7.0 (2020-09-05) draft. Now UWPX can act as a client with a propper one to one OMEMO implementation!
```
Comment (therealjeybe) [inline content/posts/newsletter/2021-05-05.md:75]: propper -> proper

**Review by therealjeybe — COMMENTED** [review]
sorry, clicked review by accident. 
just throwing my opinion on y'all while preparing the german translation

File: content/posts/newsletter/2021-05-05.md (hunk 3 above)
Comment (Echolon) [reply content/posts/newsletter/2021-05-05.md]: Thanks! 37104faa1db7f5a1d0a6a1136a8849dd8111de72

**Comment by Echolon** [comment]
### :hourglass:  **Editorial closing of the XMPP Newsletter March release** 

### :rocket: Merge in a few minutes! Docker build hopefully pulls it between 8 - 10 pm :

### :pray: Thanks to alkino, emus, Licaon_Kter, im, Martin, mathieui, MattJ, nicola, peetah, Sam, seveso, therealjeybe, wurstsalat and Ysabeau for their help in creating it!

### :handshake:   ... ehh.. here we go again, please [let it end](https://timetoherd.com/)... :hand: :mask: :hand: 

### :heart: :fireworks: Next release on 5th of June 2021

**Pull request merged**